### DISCUSSION TITLE
Blog de notícias

### DISCUSSION THREAD

**Discussion by flaviofilipe** [ifba-vca/bsi-ifba-web#20] [01 Requisitos e projeto]
A proposta seria ter um blog para postar notícias no site. Frequentemente temos informações não só institucionais mas da área como um todo: eventos, capacitações, propostas de trabalho, etc. 
Porém para isso precisaremos de pessoas para postar e gerenciar. O Backend já está preparado para fazer as postagens e já tem a API. Precisamos pensar como seria o designer do blog.
Coisas como notícias, vagas e eventos poderão fazer parte do blog.

**Comment by filipecancio** [comment]
Como está o modelo de dados do backend? posso estar formatando os componentes para recebe-lo e montar uma camada de serviço ✌🏽

**Reply by flaviofilipe** [reply]
Essas são os endpoints do Backend:
https://github.com/ifba-vca/bsi-ifba-backend/blob/master/blog/urls.py

Esses são os modelos do BD:
https://github.com/ifba-vca/bsi-ifba-backend/blob/master/blog/models.py

Vc pode baixar e acessar a rota principal no brawser pra testar os endpoints lá.

A API só servirá para listar os dados. O cadastro de novas postagens será pelo browser na rota */admin* (já está implementado com o Django Admin).

**Reply by filipecancio** [reply]
no caso, como seria o formato em .json? 
Boiei nos arquivos em phyton kkk
